File: Iniciante/URI 1002.scala
import java.util.Scanner

object Main {
    
    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val raio = leitor.nextDouble()
        val area = 3.14159 * (raio * raio)
        printf("A=%.4f\n", area)
    }
    
}

File: Iniciante/URI 1002.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val raio: Double = leitor.nextDouble();
    val area: Double = 3.14159 * (raio * raio);
    
    println(String.format("A=%.4f", area));
}